src/lib/api/configuration-shell-api/configurationShellApiInterface.ts

```diff
@@ -0,0 +1,25 @@
+import { Submodel } from '@aas-core-works/aas-core3.0-typescript/dist/types/types';
+
+export interface ConfigurationShellApiInterface {
+    getIdGenerationSettings(): Promise<Submodel>;
+
+    processGetIdGenerationSettings(response: Response): Promise<Submodel>;
+
+    putSingleIdGenerationSetting(
+        idShort: string,
+        bearerToken: string,
+        values: {
+            prefix: string;
+            dynamicPart: string;
+        },
+    ): Promise<void>;
+
+    putSingleSettingValue(
+        path: string,
+        bearerToken: string,
+        value: string,
+        settingsType: string,
+    ): Promise<void | Response>;
```

Comment:
Should either return a Response or throw. In either way, will not return void. Can and should be deleted